ui/frontend/editor/rust_monaco_def.ts

```diff
@@ -71,6 +71,9 @@ export const grammar: languages.IMonarchLanguage = {
   // The main tokenizer for our languages
   tokenizer: {
     root: [
+      [/r"/, { token: 'string.quote', next: '@rawstring0' }],
+      [/r#"/, { token: 'string.quote', next: '@rawstring1' }],
+      [/r##"/, { token: 'string.quote', next: '@rawstring2' }],
```

Comment:
Won't doing it this way only support raw strings with up to two `#`s? The ace editor supports an unlimited amount of `#`s, and I'd be amazed if this Monaco tokenizer engine wasn't built to offer up a way to match the starting `#`s to the ending `#`s.

Reply:
This ought to do it (see also second suggestion below):
```suggestion
      [/r"/, { token: 'string.quote', next: '@rawstring0' }],
      [/r(#+)"/, { token: 'string.quote', next: '@rawstring1.$1' }],
```

Reply:
I applied your suggestion, thanks!